src/gui/mzroll/main.cpp

```diff
@@ -27,11 +28,21 @@
 #include "controller.h"
 #include "elmavenlogger.h"
 
+#ifdef __OSX_AVAILABLE
```

Comment:
What's the difference between the `__OSX_AVAILABLE` and `Q_OS_MAC` macro?

Reply:
It's just that `__OSX_AVAILABLE` is independent of the Qt framework. This one is provided by the OSX native C++ headers themselves.